sys/auto_init/netif/auto_init_at86rf2xx.c

```diff
@@ -58,11 +59,19 @@ void auto_init_at86rf2xx(void)
             LOG_ERROR("[auto_init_netif] error initializing at86rf2xx radio #%u\n", i);
         }
         else {
-            gnrc_netdev_init(_at86rf2xx_stacks[i],
-                             AT86RF2XX_MAC_STACKSIZE,
-                             AT86RF2XX_MAC_PRIO,
-                             "at86rf2xx",
-                             &gnrc_adpt[i]);
```

Comment:
This is not actually deleted, it stays [here](https://github.com/RIOT-OS/RIOT/pull/6554/files/5057639f97118fd03cdb211ab5754017ae3c4f91#diff-718a84930566a0afc63ed00f0e2aeca8R69). :-)